x-pack/plugins/cloud_security_posture/server/routes/configuration/update_rules_configuration.ts

```diff
@@ -47,8 +47,8 @@ export const getPackagePolicy = async (
 export const getCspRules = async (
   soClient: SavedObjectsClientContract,
   packagePolicy: PackagePolicy
-) => {
-  const cspRules = await soClient.find<CspRuleSchema>({
+): Promise<SavedObjectsFindResponse<CspRuleSchema, unknown>> => {
+  const cspRules = soClient.find<CspRuleSchema>({
```

Comment:
nit
the current combination of async function that returns a promise is a bit messy now.
so what I think would fit here is to remove the `async` keyword at line 47 and in line 51 just do `return soClient.find<CspRuleSchema>({...`

Doing that, would ease the work for the next developer in reading this code